config/routes.rb

```diff
@@ -1,10 +1,12 @@
 Rails.application.routes.draw do
   root 'homes#index'
+
   get "/games", to: 'homes#index'
+  get "/games/:id", to: 'homes#show'
```

Comment:
We don't need to create a separate action, we can reuse the same index action through the homes controller because all we need for our react components is a div with the id `app`